profiles/tests/test_audit_log.py

```diff
@@ -79,3 +80,23 @@ def test_audit_log_create(user, enable_audit_log, enable_audit_log_username):
             "profile_id": str(profile.pk),
             "profile_part": "base profile",
         }
+
+
+def test_actor_is_resolved_in_graphql_call(
+    enable_audit_log, enable_audit_log_username, live_server, profile
+):
+    user = profile.user
+    query = """
+        query {
+            myProfile {
+                id
+            }
+        }"""
+
+    with patch_logger("audit", "info") as cm:
```

Comment:
patch_logger seems to be going the way of the Dodo.

From patch_logger function doc:
```
Use unittest.assertLogs() if you only need Python 3 support. This
    private API will be removed after Python 2 EOL in 2020 (#27753).
```

Reply:
So it seems, good catch. Replaced it with a Pytest log capture solution in 4172aee324.